electron/webpack/webpack.config.main.ts

```diff
@@ -0,0 +1,69 @@
+import type { Configuration } from 'webpack';
+import type IForkTsCheckerWebpackPlugin from 'fork-ts-checker-webpack-plugin';
+import TerserPlugin from 'terser-webpack-plugin';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const ForkTsCheckerWebpackPlugin: typeof IForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+import paths from './paths';
+
+export const mainConfig: Configuration = {
+  /**
+   * This is the main entry point for your application, it's the first file
+   * that runs in the main process.
+   */
+  entry: paths.electronMain,
+  module: {
+    rules: [
+      // Add support for native node modules
+      {
+        // We're specifying native_modules in the test because the asset relocator loader generates a
+        // "fake" .node file which is really a cjs file.
+        test: /native_modules[/\\].+\.node$/,
+        use: 'node-loader',
```

Comment:
Interesting. What happens if something we pull in uses an actual native `.node` file? Will this break?

Reply:
This is boilerplate forge config (see https://www.electronforge.io/config/plugins/auto-unpack-natives) but since it is unlikely that we will need native node modules, I removed this for now.